docs/changelog.rst

```diff
@@ -19,6 +19,14 @@ Changes in 1.3.x
    ``Cl(6)`` is 20\ |times| faster. This is achieved by deferring product JIT
    compilation until the product is used for the first time.
  * A new :mod:`clifford.transformations` module for linear transformations.
+ * Two new :doc:`predefined-algebras`, :mod:`clifford.dpga` and :mod:`clifford.dg3c`.
+ * Additional testing and assorted improvements for :mod:`clifford.tools.g3c`
+ * :func:`clifford.tools.g3c.closest_point_on_circle_from_line` has now been implemented
+   roughly following the proceedure described in Appendix A of Andreas Aristidou's PhD thesis
```

Comment:
```suggestion
   roughly following the procedure described in Appendix A of Andreas Aristidou's PhD thesis
```
A link to either the thesis or a record of it would be good here.